src/modules/dtfs-storage/dtfs-storage-file.service.ts

```diff
@@ -0,0 +1,34 @@
+import { FileGetPropertiesResponse, ShareFileClient, StorageSharedKeyCredential } from '@azure/storage-file-share';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import DtfsConfig from '@ukef/config/dtfs-storage.config';
+
+import { DtfsStorageException } from './exception/dtfs-storage.exception';
+import { DtfsStorageFileNotFoundException } from './exception/dtfs-storage-file-not-found.exception';
+
+@Injectable()
+export class DtfsStorageFileService {
+  private readonly baseUrl: string;
+  private readonly storageSharedKeyCredential: StorageSharedKeyCredential;
+
+  constructor(
+    @Inject(DtfsConfig.KEY)
+    { baseUrl, accountName, accountKey }: ConfigType<typeof DtfsConfig>,
+  ) {
+    this.baseUrl = baseUrl;
+    this.storageSharedKeyCredential = new StorageSharedKeyCredential(accountName, accountKey);
+  }
+
+  async getFileProperties(fileName: string, fileLocationPath: string): Promise<FileGetPropertiesResponse> {
+    const url = `${this.baseUrl}/${fileLocationPath}/${fileName}`;
+    const options = { fileRequestIntent: 'backup' };
```

Comment:
That was my understanding as well, but I hadn't tried omitting it, and having just tried that now, it still works. So I'm happy to leave it out altogether?

Reply:
All of the sample docs I can see don't mention setting it either. Do you remember why we first thought we should set it?

Reply:
It was when I was looking at the [ShareClientConfig interface docs](https://learn.microsoft.com/en-us/javascript/api/@azure/storage-file-share/shareclientconfig?view=azure-node-latest) to see what could be passed in the (optional) options parameter to the `ShareFileClient`. The description makes it sound like it's required

Reply:
Oscar and I spoke about this on Slack. The documentation that Oscar linked makes it sound like it's required for "Azure File OAuth over REST", which is referring to the Azure AD auth method as far as we can tell. Since we are using the account key auth method instead, we don't think it applies to us, and we've tested that it works without the option. Therefore we're going to remove it